### ISSUE TITLE
Posts not loading on home or posts page

### ISSUE THREAD

**Issue by jamestagal** [jamestagal/edtechdesigner#32]
Hi @jimafisk 

After pushing some minor updates yesterday, I am not seeing post load either on the [home page](https://edtechdesigner.io/) or the post page, see screenshots below. I can only see the navigation.  
<img width="1605" alt="Screen Shot 2022-07-19 at 10 54 12 am" src="https://user-images.githubusercontent.com/6309595/179641350-700c3fb8-e038-47f3-91dd-570e077cb452.png">
and on Posts page
<img width="1605" alt="Screen Shot 2022-07-19 at 10 54 27 am" src="https://user-images.githubusercontent.com/6309595/179641442-c5e6d3ea-90f0-486a-80cb-a0e6cf11e29e.png">

The weird thing is if I click on a page, in the nav such as 1, it will load the posts otherwise it is blank. It is like as if the root is not indexing the posts...See screenshot below.
<img width="1578" alt="Screen Shot 2022-07-19 at 10 56 33 am" src="https://user-images.githubusercontent.com/6309595/179641584-377a6562-7240-4112-99c8-b5540f9f6732.png">

Also serving this site locally is fine. All content loads as expected. 
Could it be a caching issues? though i cleared my browsers cached images and files.
I wonder whether you might take a look for me, and see if you get the same behaviour please.

Ben

**Comment by jimafisk** [comment]
I expected this, I pushed out a breaking change to pagination: https://github.com/plentico/plenti/issues/191#issuecomment-1186242647. Typically I should wait to the next minor release (v0.6.0) to do this, but selfishly I needed it for a project so I just did it in the current 0.5.x (Sorry!).

Your site is still working locally because you're probably still on plenti 0.5.9. If you update to 0.5.10 locally it will break your site (but you should do this anyways because of the fix below).

I just pushed an update to the compendium theme for this: https://github.com/plenti-themes/compendium/commit/ad5210d28f77200005c2141e0b10bcdc06db0b1a

To pull this update into your project just run `plenti theme update compendium`. Then track the changes and push it up to your repo and the build should work again. Just let me know if that gives you any trouble!

**Comment by jamestagal** [comment]
Hi @jimafisk 
Thanks for your response. no worries. I updated plenti to version 0.5.10 and the compendium theme and pushed the changes back..as you outlined. Thanks.

Though that fixed the home page, and the content loads there but not on the [Posts ](https://edtechdesigner.io/posts)page. 

Ben
UPDATE
I worked it out...I am overriding the compendium ```projs.svelte``` file so I just added the change you made, see below and that worked :)
```
$: currentPage = content.pager ? content.pager : 1;
```
Ben
